libscript/src/module-bitwise.cpp

```diff
@@ -33,26 +33,35 @@ extern "C" MC_DLLEXPORT void MCBitwiseEvalBitwiseXor(integer_t p_left, integer_t
 
 extern "C" MC_DLLEXPORT void MCBitwiseEvalBitwiseNot(integer_t p_operand, integer_t& r_output)
 {
-    r_output = ~((int32_t)p_operand);
+    r_output = ~p_operand;
 }
 
-extern "C" MC_DLLEXPORT void MCBitwiseEvalBitwiseShift(integer_t p_operand, bool p_is_right, integer_t p_shift, integer_t& r_output)
+extern "C" MC_DLLEXPORT void MCBitwiseEvalBitwiseShift(integer_t p_operand, bool p_is_right, uinteger_t p_shift, integer_t& r_output)
```

Comment:
Would it better to split this into explicit ShiftLeft and ShiftRight eval functions?